### PR TITLE
fix: add <OshekharO>

### LINKED ISSUE #1711: [PROFILE ADDITION] <Saksham Shekher>
### Profile screenshot

217171649-a9cfaca7-944b-46b9-b21a-8165bf98555c

### PR DESCRIPTION
**Pull request by OshekharO**
resolves #1711 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/OshekharO.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/95137948/217157979-0b9ccbee-9415-471c-9849-d9796923c5c1.jpg",
```
Comment (Panquesito7) [inline profiles/OshekharO.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/217171649-a9cfaca7-944b-46b9-b21a-8165bf98555c.png",
```

Comment (OshekharO) [reply profiles/OshekharO.json:2]: Sure

**Review by Vyvy-vi — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by OshekharO** [comment]
?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**